src/js/components/TextInput/TextInput.js

```diff
@@ -129,6 +132,25 @@ const TextInput = forwardRef(
     );
 
     const [suggestionsAtClose, setSuggestionsAtClose] = useState();
+    const [tipContent, setTipContent] = useState(
+      format({ id: 'input.readOnlyCopyValidation', messages }),
+    );
+
+    const copyOnClick = () => {
+      global.navigator.clipboard.writeText(value);
+      announce(
+        format({ id: 'input.readOnlyCopyValidation', messages }),
+        'assertive',
+      );
+      setTipContent(format({ id: 'input.readOnlyCopyValidation', messages }));
+    };
+
+    const copyOnBlur = () => {
```

Comment:
```suggestion
    const onBlurCopy = () => {
```
similar suggestion to above

Reply:
done